codex-ui/src/vue/switch/Switch.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div
+    :class="[$style.switch, $style[`switch-${switchValue}-${disabled ? 'disabled' : 'default'}`]]"
```

Comment:
- 'default' seems redundant
- use `--` for modifiers
- `switchValue` seems redundant. Just add a class `switch--on` when switch is on and `switch--disabled` when disabled